index.html

```diff
@@ -274,7 +274,7 @@
       </section>
       <section>
         <h2>
-          <code>OrientationType</code> enum
+          <code><dfn>OrientationType</dfn></code> enum
```

Comment:
same here about `code`

Reply:
... and globally.

Reply:
@marcoscaceres Thanks for the hint, can I make new PR for these fixes?